cmd/pktvisord/main.cpp

```diff
@@ -70,7 +70,6 @@ static const char USAGE[] =
                                   Please see https://pktvisor.dev for more information
     Modules:
       --module-list               List all modules which have been loaded (builtin and dynamic)
-      --module-load FILE          Load the specified dynamic module
       --module-dir DIR            Set module search path
```

Comment:
Change description to "Set module load path. All modules in this directory will be loaded."